emails/migrations/0059_profile_last_engagement.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.7 on 2024-01-03 22:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("emails", "0058_profile_onboarding_free_state"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="profile",
+            name="last_engagement",
+            field=models.DateTimeField(blank=True, db_index=True, null=True),
```

Comment:
👍 using `blank=True, null=True` means that database-level defaults are not needed.